ledger/participant-integration-api/src/main/scala/platform/indexer/JdbcIndexer.scala

```diff
@@ -360,13 +360,12 @@ private[indexer] class JdbcIndexer private[indexer] (
           .viaMat(KillSwitches.single)(Keep.right[NotUsed, UniqueKillSwitch])
           .mapAsync(1) {
             case (offset, update) =>
-              withEnrichedLoggingContext(JdbcIndexer.contextFor(offset, update)) {
-                implicit loggingContext =>
-                  Timed
-                    .future(
-                      metrics.daml.indexer.stateUpdateProcessing,
-                      handleStateUpdate(offset, update),
-                    )
+              withEnrichedLoggingContext(JdbcIndexer.contextFor(offset, update)) { _ =>
+                Timed
+                  .future(
+                    metrics.daml.indexer.stateUpdateProcessing,
+                    handleStateUpdate(offset, update),
+                  )
```

Comment:
Same considerations in play as in https://github.com/digital-asset/daml/pull/6992/files#r465275865

Reply:
This looks like a bug. The logging context should be passed through to `handleStateUpdate`. Instead it's using one that's passed in at the class constructor level. I'll open a separate fix.

Reply:
Fixed in #7042. Please take a look when you can.

Reply:
Included in this PR's build.